docs/overview.rst

```diff
@@ -45,5 +45,27 @@ set the values of parameters for all components.
 **Postprocess results.** Results of an oemof.solph optimisation can be postprocessed and saved as
 a :class:`ResultsDataPackage <oemoflex.model.datapackage.ResultsDataPackage>`.
 
+Postprocessed results are stored as `csv` files in the following directory structure:
+
+.. code-block::
+
+    postprocessed
+    ├── sequences
+    │     ├── bus
+    │     ├── by_variable
+    │     ├── component
+    ├── objective.csv
+    ├── scalars.csv
+
+The directory `sequences` contains time series of flows. In directory `bus` flows from and to each bus can be found.
+In the directory `by_variable` a `csv` file contains all time series of the flows and another `csv` file contains the
```

Comment:
Most of the time, it is these two. Generally, it is any optimisation variable.

Reply:
Thank you for this hint. How about the change I made in commit https://github.com/rl-institut/oemoflex/pull/76/commits/ba0e0f5b5ad2e0ec2c4ac4e4f4a56e5f7b0f1339?

Reply:
Better! Thanks.